website/addons/box/routes.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+"""Box addon routes."""
+from framework.routing import Rule, json_renderer
+
+from website.addons.box import views
+
+
+auth_routes = {
+    'rules': [
+
+        Rule(
+            '/settings/box/',
+            'get',
+            views.auth.box_user_config_get,
+            json_renderer,
+        ),
+
+        ##### OAuth #####
+        Rule(
+            '/settings/box/oauth/',
+            'get',
+            views.auth.box_oauth_start,  # Use same view func as node oauth start
+            json_renderer,
+            endpoint_suffix='_user'          # but add a suffix for url_for
+        ),
+
+        Rule(
+            '/addons/box/oauth/finish/',
```

Comment:
For consistency with @lyndsysimon 's Oauth system which we will likely migrate to in the near future, let's use `/oauth/callback/box/`.